project-base/config/services.yaml

```diff
@@ -11,6 +11,8 @@ services:
         autowire: true
         autoconfigure: true
         public: false
+        bind:
+            $secret: '%secret%'
```

Comment:
Why is this needed? I thought the secret is obtained from ParameterBag

Reply:
c1c034f8fb112803cd13dbc0dd0f9fd5a767394d